src/styles/blocks/catalog.scss

```
@@ -17,6 +18,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms;
+    transition-duration: 300ms;
```

You already set duration, also you change color< so please add it.
make same for header
```suggestion
    transition: transform 0.3s, color 0.3s;
```